src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/mapper/PersonopplysningerMapper.kt

```diff
@@ -142,6 +146,8 @@ class PersonopplysningerMapper(
             },
             adresse = barn.bostedsadresse.map(adresseMapper::tilAdresse),
             borHosSøker = AdresseHjelper.borPåSammeAdresse(barn, bostedsadresserForelder),
+            deltBosted = barn.deltBosted.filter { !it.metadata.historisk },
```

Comment:
Hvorfor filter på !historisk ?

Reply:
Vi tolker det som at historiske perioder er "ugyldige", og filtrerer de ut for å kun ta med de som er gyldige (som bare kanskje er én?). Ikke sikker på om det er best å ta med alle historiske, eller bare muligens den ene gyldige / ikke-historiske.

Reply:
Ja, jeg som surret, sikkert fint. lage en `.gjeldende()` :)